typescript/codepipeline-build-deploy/lib/codepipeline-build-deploy-stack.ts

```diff
@@ -224,11 +249,13 @@ export class CodepipelineBuildDeployStack extends cdk.Stack {
     const sourceStage = {
       stageName: "Source",
       actions: [
-        new pipelineactions.CodeCommitSourceAction({
-          actionName: "AppCodeCommit",
-          branch: "main",
+        new githubactions.GitHubSourceAction({
+          actionName: "GitHub",
+          owner: 'owner_name',
+          repo: 'repo_name',
+          oauthToken: cdk.SecretValue.secretsManager('github-oauth-token'),
```

Comment:
Please document that the user should update this value or create it using a specific name.